### PR TITLE
Feature/94 ants in streamerpage

### PR DESCRIPTION
**Pull request by AlexAnarcho**
Reworked the StreamerPage to work with Antd. Multiple Components removed because they are no longer necessary.
- Nav
- CategoryNav

### CODE REVIEW

File: src/components/StreamerCard.jsx [hunk 1]
```diff
@@ -1,83 +1,71 @@
...
+    <Card
+      cover={
+        <img
+          alt="example"
+          /* src={streamer.profilePicture} */
+          src="https://i.imgur.com/PW3XO3u.jpg"
+        />
+      }
+      actions={[<TipButton streamerName={streamer.userName} />]}
+      style={{ textAlign: "center", maxWidth: "300px", minWidth: "100px" }}
```
Comment (hundehausen) [inline src/components/StreamerCard.jsx:51]: export style

**Review by hundehausen — CHANGES_REQUESTED** [review]

**Comment by AlexAnarcho** [comment]
Created a new directory for StreamerPage with StreamerCard and index.less

File: src/pages/Donate/index.jsx [hunk 2]
```diff
@@ -7,6 +7,7 @@ import NotFound404 from "../NotFound404";
 import Success from "./Success";
 import { Row, Col, Switch } from "antd";
 import "./index.less";
```
Comment (hundehausen) [inline src/pages/Donate/index.jsx:9]: rename do Donate.less. Rename all .less files in componenents folders to <FolderName>.less so it doesn't mix up with others index.less files.

File: src/pages/Donate/Success.jsx [hunk 3]
```diff
@@ -1,6 +1,7 @@
 import PropTypes from "prop-types";
 import { Row, Col, Typography, Button } from "antd";
 import "./index.less";
```
Comment (hundehausen) [inline src/pages/Donate/Success.jsx:3]: rename do Donate.less. Rename all .less files in componenents folders to .less so it doesn't mix up with others index.less files.

File: src/pages/Donate/Payment.jsx [hunk 4]
```diff
@@ -4,6 +4,7 @@ import monerojs from "../../libs/monero";
 import { Row, Col, Spin, Typography, Image, Divider } from "antd";
 import { WalletOutlined } from "@ant-design/icons";
 import "./index.less";
```
Comment (hundehausen) [inline src/pages/Donate/Payment.jsx:6]: rename do Donate.less. Rename all .less files in componenents folders to .less so it doesn't mix up with others index.less files.

File: src/pages/Donate/EnterMessage.jsx [hunk 5]
```diff
@@ -15,6 +15,7 @@ import {
 } from "antd";
 import { DesktopOutlined } from "@ant-design/icons";
 import "./index.less";
```
Comment (hundehausen) [inline src/pages/Donate/EnterMessage.jsx:17]: rename do Donate.less. Rename all .less files in componenents folders to .less so it doesn't mix up with others index.less files.

File: src/pages/StreamerPage/StreamerCard.jsx [hunk 6]
```diff
@@ -0,0 +1,75 @@
...
+import tipxmr from "../images/tipxmr-button.png";
+import { Link } from "react-router-dom";
+import { Row, Col, Card, Image, Tag, Typography } from "antd";
+import {
+  EditOutlined,
+  EllipsisOutlined,
+  GiftOutlined,
+} from "@ant-design/icons";
+import "../styles/index.less";
+import "./index.less";
```
Comment (hundehausen) [inline src/pages/StreamerPage/StreamerCard.jsx:12]: rename do StreamerPage.less. Rename all .less files in componenents folders to .less so it doesn't mix up with others index.less files.

File: src/pages/StreamerPage/index.jsx [hunk 7]
```diff
@@ -0,0 +1,77 @@
+import { useEffect, useState } from "react";
+import StreamerCard from "./";
+import socketio from "../../libs/socket_donator";
+import { Row, Col, Layout, Menu, Typography } from "antd";
+import "../../styles/index.less";
+import "./index.less";
```
Comment (hundehausen) [inline src/pages/StreamerPage/index.jsx:6]: rename do StreamerPage.less. Rename all .less files in componenents folders to .less so it doesn't mix up with others index.less files.

**Review by hundehausen — CHANGES_REQUESTED** [review]

**Review by hundehausen — APPROVED** [review]

**Pull request merged**